pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -136,10 +137,9 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 					// we will flag the account with the Failed Reuse condition, and with state = Failed
 
 					// First we want to see if this was an update race condition where the credentials rotator will update the CR while the finalizer is trying to run.  If that's the case, we want to requeue and retry, before outright failing the account.
-					if errors.IsConflict(err) {
+					if k8serr.IsConflict(err) {
 						reqLogger.Info("Account CR Modified during CR reset.")
-						err = fmt.Errorf("Account CR Modified during CR reset. %s", err.Error())
-						return reconcile.Result{}, err
+						return reconcile.Result{}, errors.Wrap(err, "account CR modified during reset")
```

Comment:
Looks like you need to update the expected string in the test (or change this one back).

Reply:
I saw that, fixed.  Thanks!